modules/ROOT/pages/6.4.1-release-notes.adoc

```diff
@@ -0,0 +1,1329 @@
+
+To fix this issue, {productname} added improvements to the list configuration to checks a new set of conditionals which would determine if the `node` selected matched the parent `node` within the checklist. If the match returns as `true`, the (api) will set only the checklist icon to a `active` state.
+
+As a result, the checklist `icon` in {productname} 6.4.1, related to the `advlist` plugin will only appear in a `active` state, when the `node` checklist meets the new conditions implemented.
+
+=== Dragging transparent elements into transparent blocks elements could produce invalid nesting of transparents
+//#TINY-9231
+
+Dragging or dropping block `links` into other `links` was producing invalid nesting of other transparents such as `anchors`. As a result, these transparent elements would produce `anchor` tags within other `anchor` tags, which is not a general practice. As an example `links` and `anchors` that are defined to an element, must not be nested within or contain any other element and vice versa.
+
+Changes were implemented in {productname} 6.4.1 to fix this issue to ensure that any invalid child anchor elements that try to reproduce this action are unwrapped when the editor verifies another transparent has been dropped into another transparent (anchor) element.
+
+In {productname} 6.4.1, dragging content such as `links` into other `links` will now produce *valid* HTML `output` after a drag operation occurs which will be consistent across all browsers.
+
+=== The editor.insertContent API would insert contents inside noneditable elements if the selection was inside the element
+//#TINY-9462
+
+In previous versions of {productname}, the settings for the `contentEditable` property in the editor root was set to `false`. As a consequence, the user could insert content using `api` commands to edit a `contenteditable="false"` root element.
+
+To fix this bug, in {productname} 6.4.1, a blocker was added to the `editor.insertContent` API and `mceInsertContent` command, which stops `contenteditable="false"` root elements from being edited.
+
+In {productname} 6.4.1, inserting contents inside a `contenteditable="false"` element while the selection is inside another element, is no longer possible.
+
+=== Closing a dialog would scroll down the document in Safari
+//#TINY-9148
+
+While using previous versions of {productname}, `clicking` away to dismiss the *Search* and *Replace* dialog within Safari would scroll the page to the bottom of the editor. As a result the user would have to manually scroll back to the original `focus` area each time the steps above were executed.
+
+To correct this issue, a new Css.set was implemented to keep the `focus` on the DOM container in the initial scroll document position. 
```

Comment:
```suggestion
To correct this issue, the active element will be blurred before closing the dialog.
```

Maybe? Safari scrolls down to the sink if the dialog is selected before removing the dialog, so the changes made in the ticket were blurring the active element. Then remove the dialog so it doesn't scroll.